### PR TITLE
[tests-only][full-ci] Add test coverage for correct parent folder reference in activity messages

### PR DESCRIPTION
**Pull request by anon-pradip**
## Description

This PR adds test coverage to ensure that activity messages correctly reference the parent folder for file operations

## Related Issue

- Test coverage for bug: https://github.com/owncloud/ocis/issues/9536

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiActivities/activities.feature [hunk 1]
```diff
@@ -3930,3 +3930,246 @@ Feature: check activities
         }
       }
       """
+
+  @issue-9536
+  Scenario: check activities after moving a file inside subfolder inside project space
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:3935]:
```suggestion
  Scenario: check activities after moving a file inside a subfolder in a project space
```

File: tests/acceptance/features/apiActivities/activities.feature [hunk 2]
```diff
@@ -3930,3 +3930,246 @@ Feature: check activities
...
+                              }
+                            }
+                          }
+                        }
+                      }
+                    },
+                    "times": {
+                      "type": "object",
+                      "required": ["recordedTime"]
+                    }
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:4167]: you can remove this part

File: tests/acceptance/features/apiActivities/activities.feature [hunk 3]
```diff
@@ -3930,3 +3930,246 @@ Feature: check activities
...
+  @issue-9536
+  Scenario: check activities after moving a file inside a subfolder in a project space
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has created a folder "/folder" in space "new-space"
+    And user "Alice" has created a folder "/folder/subfolder" in space "new-space"
+    And user "Alice" has uploaded a file inside space "new-space" with content "some content" to "/folder/test.md"
+    And user "Alice" has moved file "/folder/test.md" to "/folder/subfolder/test.md" in space "new-space"
+    When user "Alice" lists the activities of folder "/folder" from space "new-space" using the Graph API
```
[the remark this answers is not in the document]
Comment (anon-pradip) [reply tests/acceptance/features/apiActivities/activities.feature]: Got it. :rocket:

File: tests/acceptance/features/apiActivities/activities.feature [hunk 4]
```diff
@@ -3930,3 +3930,242 @@ Feature: check activities
         }
       }
       """
+
+  @issue-9536
+  Scenario: check activities after moving a file inside a subfolder in a project space
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:3935]: do we have tests for personal space?

Comment (anon-pradip) [reply tests/acceptance/features/apiActivities/activities.feature:3935]: Not yet for this same journey for Personal space. Should I write tests for Personal space too?

Comment (PrajwolAmatya) [reply tests/acceptance/features/apiActivities/activities.feature:3935]: I think following scenario covers this for personal space
https://github.com/owncloud/ocis/blob/092136baace55cf2727ad8b8d929de3e68310a30/tests/acceptance/features/apiActivities/activities.feature#L1839

Comment (amrita-shrestha) [reply tests/acceptance/features/apiActivities/activities.feature:3935]: > I think following scenario covers this for personal space
> 
> https://github.com/owncloud/ocis/blob/092136baace55cf2727ad8b8d929de3e68310a30/tests/acceptance/features/apiActivities/activities.feature#L1839

if so then can we combine this two scenarion in one by using Scenario Outline approach @anon-pradip

Comment (anon-pradip) [reply tests/acceptance/features/apiActivities/activities.feature:3935]: > > I think following scenario covers this for personal space
> > https://github.com/owncloud/ocis/blob/092136baace55cf2727ad8b8d929de3e68310a30/tests/acceptance/features/apiActivities/activities.feature#L1839
> 
> if so then can we combine this two scenarion in one by using Scenario Outline approach @anon-pradip

Doing so, I think, we will not meet the exact journey mentioned in the issue

Comment (saw-jan) [reply tests/acceptance/features/apiActivities/activities.feature:3935]: this scenario is for project space. looks like we already have test scenario for personal space

Comment (saw-jan) [reply tests/acceptance/features/apiActivities/activities.feature:3935]: we should keep this scenario.

**Review by saw-jan — APPROVED** [review]

**Review by amrita-shrestha — APPROVED** [review]

**Pull request merged**